DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -2031,6 +2033,7 @@ globus_l_gfs_iRODS_recv(
     // the following conditions are met:
     // 1. The transfer_info->alloc_size is defined (not zero) and greater than zero.
     // 2. The alloc_size < irods_parallel_file_size_threshold_bytes.
+    // 3. Issue 101 - If filename has apostrophe then force one thread.
```

Comment:
Please create an issue for the "real" fix and mention it here so that we remember to remove this comment later along with the code forcing the single-threaded transfer.

Reply:
I am unsure if you are referring to #104 or https://github.com/irods/irods/issues/3902.

I will add a comment referencing both.